packages/visx-xychart/src/components/axis/BaseAxis.tsx

```diff
@@ -66,7 +66,11 @@ export default function BaseAxis<Scale extends AxisScale>({
       ? (width ?? 0) - (margin?.right ?? 0)
       : 0;
 
-  return (
+  const scale = (orientation === 'left' || orientation === 'right' ? yScale : xScale) as
```

Comment:
Would this look cleaner?
`const scale: Scale | undefined = ...`

Reply:
Definitely ideal, but the type won't work without a coercion either here, or when I get the scales from `DataContext` (I haven't successfully figured out how to use generics with `Context` types; most Series coerce the `Context` values right now)